nncf/common/graph/graph.py

```diff
@@ -700,3 +702,15 @@ def remove_nodes_from(self, nodes: List[NNCFNode]) -> None:
         self._node_id_to_key_dict = {}
         for node_key, node in self._nx_graph.nodes.items():
             self._node_id_to_key_dict[node["id"]] = node_key
+
+    def find_patterns(self, patterns: GraphPattern) -> List[str]:
```

Comment:
Also, the name of the method is not obvious since it returns only names of layers in found patterns, not patterns as is.

Reply:
Change name